Jenkinsfile

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/groovy
-@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+@Library('github.com/hectorj2f/fabric8-pipeline-library@wit_pipeline')
```

Comment:
that happened bc I was trying a different workflow, so I modified in my own branch ;).